partner_location_auto_create/hooks.py

```diff
@@ -0,0 +1,24 @@
+# © 2018 brain-tec AG (Kumar Aberer <kumar.aberer@braintec-group.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, SUPERUSER_ID
```

Comment:
Not sure if this is needed anymore. We have a sudo now.

Reply:
It's needed to create a new Environment for the installation hook. Sudo does not help here